config/database.yml

```diff
@@ -23,7 +23,7 @@ test:
 production:
   <<: *default
   adapter: mysql2
-  encondig: utf-8
+  encoding: utf-8
```

Comment:
Please change value to 'utf8' as defined in https://github.com/brianmario/mysql2/blob/master/lib/mysql2/client.rb#L47. Apparently, production config is falling back to default value when none is passed, e.g., 'encondig' key.